.gitignore

```diff
@@ -125,3 +125,5 @@ server/migrations/test_plan_target_id.csv
 
 # Private Key files (installed by deploy)
 jwt-signing-key.pem
+
+client/resources
```

Comment:
Since there are still updates to `client/resources` in this PR, you'll also need to untrack `client/resources` by doing: `git rm -r --cached client/resources`, and push again afterwards.

Although, I am now wondering if this should actually needs to be done? It will require that contributors be *somewhat* aware of when last changes to the `w3c/aria-at` files have happened so they may re-run the import script locally. Otherwise there could be confusion if some `w3c/aria-at`-dependent operations aren't working as expected if the client application's files are updated but not the now ignored aria-at harness changes.

My above concern is relatively minor though and seems like a `docs/local-development.md` update could be useful here.

Reply:
I added a brief notes in the docs.

Reply:
Looks good! After untracking the `client/resources` folder, this PR should be good to go.

Reply:
All set!